osu.Game.Resources/ResourceAssembly.cs

```diff
@@ -0,0 +1,9 @@
+using System.Reflection;
+
+namespace osu.Game.Resources
+{
+    public static class ResourceAssembly
```

Comment:
Can we call this `OsuResources` for clarity?

Reply:
`OsuResources.ResourceAssembly`, makes sense